src/shared/Microsoft.Git.CredentialManager/Microsoft.Git.CredentialManager.csproj

```diff
@@ -16,7 +16,6 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="LibGit2Sharp.NativeBinaries" Version="2.0.278" />
     <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
```

Comment:
In the future maybe we can replace / upgrade with `System.Text.Json` 🥳

Reply:
I looked at this, but at the time there were some issues with supporting this on .NET Framework too (which we use for GCM "Core" on Windows - and we target .NET Framework on Windows because of [MSAL limitations](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1398).)

Reply:
Oh MSAL.